conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -83,7 +106,23 @@ class GramineEnclaveHandle(
         }
     }
 
-    private fun unzipIntoWorkingDir() {
+    private fun getManifestFromUnzippedBundle(): Path {
+        check(enclaveMode != EnclaveMode.MOCK)
+
+        return if (enclaveMode == EnclaveMode.SIMULATION) {
+            require((workingDirectory / GRAMINE_MANIFEST).exists()) { "Missing Gramine manifest" }
+            require((workingDirectory / GRAMINE_ENCLAVE_JAR).exists()) { "Missing enclave jar" }
```

Comment:
We can avoid the duplicate check below

Reply:
Ok, I can only save a line of code for the enclave jar, is this what you mean?

Reply:
Yep.